tests/format.html

```diff
@@ -973,3 +973,6 @@
     ORDER BY
       hire_date
   );
+---
+SELECT
+  1 :: text;
```

Comment:
TBH i do not like this formatting. most of the tools use `1::text` with no extra spaces.